From 3481748772adce09dd57c7ca5f006d7c7f37006d Mon Sep 17 00:00:00 2001
From: David Jarvie <djarvie@kde.org>
Date: Fri, 8 Mar 2013 18:00:04 +0000
Subject: [PATCH 01/33] Make it compile with gcc 4.4

The compilation fails with an error complaining about using 'typename'
outside a template.
---
 agents/nepomukfeeder/nepomukfeederagent.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agents/nepomukfeeder/nepomukfeederagent.cpp b/agents/nepomukfeeder/nepomukfeederagent.cpp
index 7004d08..dd065a0 100644
--- a/agents/nepomukfeeder/nepomukfeederagent.cpp
+++ b/agents/nepomukfeeder/nepomukfeederagent.cpp
@@ -60,7 +60,7 @@
 #include "nepomukfeeder-config.h"
 #include "nepomukfeederadaptor.h"
 
-typedef QSharedPointer< QMultiHash< typename Akonadi::Collection::Id,  typename Akonadi::Item::Id> > MultiHashPointer;
+typedef QSharedPointer< QMultiHash< Akonadi::Collection::Id,  Akonadi::Item::Id> > MultiHashPointer;
 Q_DECLARE_METATYPE(MultiHashPointer)
 
 namespace Akonadi {
-- 
1.8.1.4

